src/components/CategoryChips/CategoryChips.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useFilter } from "../../context/Filter/FilterContext";
+import { useUserDetails } from "../../context/User/UserContext";
+import { useVideo } from "../../context/Video/VideoContext";
```

Comment:
Can add index.js for exports in context folder.

Reply:
Yes, Will do that in the next PR.